tests/test_cookiejar.py

```diff
@@ -756,3 +756,36 @@ async def test_cookie_jar_clear_domain() -> None:
     assert morsel.value == "bar"
     with pytest.raises(StopIteration):
         next(iterator)
+
+
+async def test_secure_cookie_not_filtered_from_unsafe_cookiejar_when_given_unsecured_endpoint() -> None:
```

Comment:
unsecured -> insecure

Reply:
I chose the word unsecured as it has the following meaning, which insecure seems not to
> not made secure or safe.

Reply:
Yeah, but insecure is more standard. insecure means not secure, liable to risk or danger. HTTP connections are inherently insecure, not just unsecured.